### PR TITLE
Alert Application function #61

### PR DESCRIPTION
**Pull request by qbitza**
This introduces an Application alert function, resolving #61

### CODE REVIEW

**Comment by reisraff** [comment]
It doesn't work for me.

**Comment by qbitza** [comment]
@reisraff did you recompile the Lazarus project?

**Comment by reisraff** [comment]
Sorry guys, I had not compiled the lazarus... It's working pretty

**Comment by reisraff** [comment]
:+1:

**Comment by rodrigowbazevedo** [comment]
This way is not possible to pass a dialog title. maybe it'll be better if the `alert`  method receives 2 parameters, `mesasge` and `title` and both strings.

**Comment by qbitza** [comment]
That could work. I'll modify and resubmit in a bit.

File: src/Application.php [hunk 1]
```diff
@@ -382,18 +382,19 @@ public function getLoop()
...
-    public function alert($message)
+    public function alert($message, $title = '')
     {
         if (is_array($message)) {
             $message = implode('\n', $message);
         }
         $this->sendCommand(
             'showMessage',
-            array($message),
+            array($message, $title),
```
Comment (reisraff) [inline src/Application.php]: I guess you can change here from `array()` to `[]`

**Comment by qbitza** [comment]
Are you guys happy with the implementation or do you need me to make more adjustments?
@gabrielrcouto @reisraff  @rodrigowbazevedo

**Comment by gabrielrcouto** [comment]
Happy! :smile: 
Thank you @qbitza

**Pull request merged**